src/app/examples/todos/todos.reducer.spec.ts

```diff
@@ -0,0 +1,60 @@
+import * as fromTodo from './todos.reducer';
+
+describe('TodoReducer', () => {
+  describe('undefined action', () => {
+    it('should return the default state', () => {
+      const { initialState } = fromTodo;
+      const action = {} as any;
+      const state = fromTodo.todosReducer(undefined, action);
+
+      expect(state).toBe(initialState);
+    });
+  });
+
+  describe('ADD action', () => {
+    it('should add a todo', () => {
+      const { initialState } = fromTodo;
+      const action = new fromTodo.ActionTodosAdd({ name: 'Mercuccio' });
+      const state = fromTodo.todosReducer(initialState, action);
+
+      expect(state.items.length).toEqual(initialState.items.length + 1);
```

Comment:
For the tests lets use `{}` as initial state so that we can check for `items.length` that equals 1 and also for the `name` of the added todo